server/server.go

```diff
@@ -223,6 +225,10 @@ func (s *Server) RevokeUser(userID string) error {
 	return nil
 }
 
+func (s *Server) SetSystemLabelFilter(filter SystemLabelFilter) {
```

Comment:
Given we have this as a build option, i'd prefer we didn't have this as a method as well. I don't think it makes much sense to be able to configure the system label filter after the server has already been initialized. Would you mind removing this method? Or do you really need it for some reason?

Reply:
It's needed for testing bridge: 
- Environment with a server without scheduled send.
- Bridge with fully synced user.
- Server gets the Scheduled send label.
- Bridge user logs out / logs in and the scheduled send label is correctly added.

Reply:
Ah so the scheduled send label is only visible if there are messages in there? Otherwise it's hidden for the user?

Reply:
Resolved on slack